apps/frontend/src/app/5_pages/StakePage/components/PersonalStakingStatistics/PersonalStakingStatistics.tsx

```diff
@@ -0,0 +1,120 @@
+    const totalVotingPowerDecimal = decimalic(totalVotingPower.toString());
+    const getVotingPowerShare = votingPowerDecimal
+      .div(totalVotingPowerDecimal)
+      .mul(100);
+    return getVotingPowerShare.toNumber();
+  }, [votingPower, totalVotingPower]);
+
+  return (
+    <div className="w-full bg-gray-90 md:bg-gray-90 md:py-6 p-4 md:px-6 rounded mb-6">
+      <Heading
+        children={t(translations.stakePage.personalStatistics.title)}
+        className="text-base md:text-2xl font-medium mb-6"
+      />
+
+      <div className="flex flex-wrap lg:gap-16 gap-6 items-center">
+        <div className="w-full md:w-auto">
+          <PersonalStatRender
+            label={t(translations.stakePage.personalStatistics.stakedSov)}
+            value={
+              <AmountRenderer
+                value={fromWei(stakedValue)}
+                suffix={SupportedTokens.sov}
+                precision={TOKEN_RENDER_PRECISION}
+              />
+            }
+            className="text-[2rem]"
+          />
+        </div>
+        <PersonalStatRender
+          label={
+            <span className="flex items-center gap-1">
+              {t(translations.stakePage.personalStatistics.votingPower)}{' '}
+              <HelperButton
+                tooltipClassName="max-w-56 md:max-w-96"
+                content={
+                  <div className="flex flex-col">
+                    <div>
+                      {t(
+                        translations.stakePage.personalStatistics
+                          .votingPowerInfo.line1,
+                      )}
+                      :{' '}
+                      <b>
+                        {fromWei(votingPower)} {VP}
+                      </b>
+                    </div>
+                    <div>
+                      {t(
+                        translations.stakePage.personalStatistics
+                          .votingPowerInfo.line2,
+                      )}
+                      :{' '}
+                      <b>
+                        {fromWei(0)} {VP}
+                      </b>
+                    </div>
+                    <div>
+                      {t(
+                        translations.stakePage.personalStatistics
+                          .votingPowerInfo.line3,
+                      )}
+                      :{' '}
+                      <b>
+                        {fromWei(votingPower)} {VP}
+                      </b>
+                    </div>
+                  </div>
+                }
+              />
+            </span>
+          }
+          value={
+            <AmountRenderer
+              value={fromWei(votingPower)}
+              suffix={VP}
+              precision={TOKEN_RENDER_PRECISION}
+            />
+          }
+        />
+        <PersonalStatRender
+          label={t(translations.stakePage.personalStatistics.votingPowerShare)}
+          value={
+            <AmountRenderer value={votingPowerShare} suffix="%" precision={2} />
```

Comment:
```suggestion
            <AmountRenderer value={votingPowerShare} suffix="%" precision={5} />
```
It should be `5` according to the design.